### DISCUSSION TITLE
Usage with FreeIPA 4.7+ (ACME)

### DISCUSSION THREAD

**Discussion by zdzichu** [guilhem/freeipa-issuer#13] [General]
Hi,

FreeIPA 4.7+ includes ACME addon. In theory, it would make it work with plain `cert-manager`.
How do you see the future of `freeipa-issuer`?

**Comment by guilhem** [comment]
@zdzichu indeed, if you are using FreeIPA 4.7+, I think you should use ACME :)
ACME is an excellent protocol and with a far better ecosystem than the native FreeIPA library.
